### ISSUE TITLE
Type errors when following the blog article 

### ISSUE THREAD

**Issue by lpoulter1** [nexxeln/guestbook-blog#7]
Hi the code runs fine but I get the following type errors, any ideas why? 

I pushed my code [here](https://github.com/lpoulter1/guestbook).

<img width="779" alt="image" src="https://user-images.githubusercontent.com/32073440/214621055-d4ca50f7-7a78-45ad-920b-9800fc7342e0.png">

**Comment by ononotofu** [comment]
Seeing the same linting errors.

**Comment by Ruslanmsv** [comment]
@ononotofu @lpoulter1 

The problem is in Ts Server not updating types. 

You have to manually restart TS Server by restarting VS Code.

https://github.com/t3-oss/create-t3-app/issues/1224
https://github.com/prisma/prisma/discussions/3474
https://github.com/prisma/prisma/issues/14722

**Comment by PaulHaze** [comment]
> @ononotofu @lpoulter1
> 
> The problem is in Ts Server not updating types.
> 
> You have to manually restart TS Server by restarting VS Code.

Tried that many times but I get LOTS of type errors when following along, mainly:
        '@typescript-eslint/no-unsafe-call'
        '@typescript-eslint/no-unsafe-member-access'
        '@typescript-eslint/no-unsafe-assignment'
        '@typescript-eslint/no-unsafe-return'

**Comment by nexxeln** [comment]
Try restarting eslint server maybe?

**Comment by PaulHaze** [comment]
@nexxeln Yeah tried that, tried reloading the window, restarting the TS server, and the good ol' quit-vs-code-and-start-again but it didn't help. I am not an expert on T3 (hence why I was following your tutorial) so I can't comment as to what has changed, but something has shifted in the latest create-t3-app since you made your tutorial.

Give it a go with the latest t3-app and see if you hit the same problems as I and others have done....

Thanks for your efforts though! I don't want to seem like I'm having a dig at you :)
